### ISSUE TITLE
Add about disableStorageReset parameter to README

### ISSUE THREAD

**Issue by AleksandrHorev** [abhinaba-ghosh/playwright-lighthouse#5]
Hi!
I have tried to use your library and it works very well.
One suggestion, could you please add to readme an example about *disableStorageReset* parameter.
Thank you 

[README.txt](https://github.com/abhinaba-ghosh/playwright-lighthouse/files/5084462/README.txt)

**Comment by abhinaba-ghosh** [comment]
Hi @AleksandrHorev, Thanks for raising the PR. Are you referring to: https://github.com/GoogleChrome/lighthouse/issues/2599#issuecomment-523057473?

@tabermike as you are using this libarry for long. Have you came across this scenario? In case you did, any solution you figured out?

**Comment by AleksandrHorev** [comment]
Hi @abhinaba-ghosh ,
I have used this way in my investigations. So, no, I didn't referring on anything :)
The idea why I suggested it:
I tested a website with lighthouse by dev tool manually and compared it with results from playwright-lighthouse. The results were different and I investigated the reason for it, it was a *disableStorageReset* parameter.
I suggested it would be helpful to share it in the documentation.
